pkg/generators/tags/tags.go

```diff
@@ -7,26 +7,39 @@ import (
 )
 
 const (
-	Name    = "kanopy:builder"
-	Package = "package"
+	Builder         = "kanopy:builder"
+	Builder_Package = "package"
+	Builder_OptIn   = "true"
+	Builder_OptOut  = "false"
+	Receiver        = "kanopy:receiver"
+	Receiver_Ptr    = "ptr"
+	Receiver_Value  = "value"
```

Comment:
Happy to take suggestions on naming of these constants.
Convention here is `<tag>_<value option>`, but can change.

Reply:
Only nit I have is change `ptr` to `pointer`